### PR TITLE
Merchant item edit page

### PR DESCRIPTION
**Pull request by MadelineMauser**
- Added the merchant item edit page. 
- Updated the items controller update action to accommodate the editing of an item. 
- Added strong params for items.
- Added a link to the merchant item show page that redirects to that item's edit page
- Added flash message implementation to the application layout

### CODE REVIEW

**Review by kenzjoy — APPROVED** [review]
approved!

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -5,20 +5,43 @@ def index
...
 
+  def edit 
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = Item.find(params[:id])
+  end
+
   def update
     item = Item.find(params[:id])
-    if params[:button] == 'true' && item.status == 'enabled'
+    if params[:commit] == 'Update'
```
Comment (msakr21) [inline app/controllers/items_controller.rb:20]: I love this change! Pretty smart!

File: app/views/items/show.html.erb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <p>Item: <%=@item.name%></p>
 <p>Description: <%=@item.description%></p>
-<p>Current price: $<%="#{@item_price}".insert(-3, '.')%></p>
\ No newline at end of file
+<p>Current price: $<%="#{@item_price}".insert(-3, '.')%></p>
```
Comment (msakr21) [inline app/views/items/show.html.erb:3]: Very curious what this line does! Would like to meet up at some point today so you'd teach me if you're willing to!

**Review by msakr21 — APPROVED** [review]
Looks good. I submitted some feedback(acknowledgment and a request)

**Comment by msakr21** [comment]
Going to go ahead and merge this so we can pull a unified main before working on further user stories.

**Pull request merged**